Libraries/DateTime/NodaTimeExtensions.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;

```

Comment:
Ebből van egy csomó, ez mi is, még egyszer?

Reply:
Encoding változás, nem UTF-8 volt a fájl.